test/TritonGPU/combine.mlir

```diff
@@ -1568,219 +913,12 @@ module attributes {"triton_gpu.num-warps" = 4 : i32, "triton_gpu.threads-per-war
   }
 }
 
-
-// -----
-
-// Check that we don't have extra convert for flash attention IR.
```

Comment:
Same as above I don't think we should remove this test

Reply:
Fixed manually.